src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -527,6 +528,11 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
 
                     // Clear cached analyzer descriptors and unregister exception handlers hooked up to the LocalizableString fields of the associated descriptors.
                     analyzerManager.ClearAnalyzerState(analyzers);
+
+                    if (Arguments.ReportAnalyzer)
+                    {
+                        ReportAnalyzerExecutionTime(consoleOutput, analyzerDriver, compilation.Options.ConcurrentBuild);
```

Comment:
analyzerDriver can be null here if execution enters the finally handler because an exception occurs before the creation of the driver completes. Maybe we don't care, or maybe there should be a check for null here.